scripts/dev

```diff
@@ -136,6 +137,23 @@ task :down do
   sh "docker-compose down"
 end
 
+desc "Restart the specified container"
+task restart: :consume_args do |t, args|
+  yml = `docker-compose config`
+  config = YAML.load(yml)["services"]
+  services = config.keys.sort
+
+  args.to_a.each do |service|
+    unless services.include?(service)
+      puts "service #{service} not one of defined services: #{services.join(', ')}"
+      next
+    end
+    sh "docker-compose rm -s --force #{service}"
+  end
+
+  Rake::Task["up"].invoke
```

Comment:
I think maybe we should target just the service we killed here? OTOH, most of the stuff in this file assumes that all the containers are running...